cmd/plume/fedora.go

```diff
@@ -102,6 +105,13 @@ var (
 	}
 )
 
+func AddFedoraSpecFlags(flags *pflag.FlagSet) {
```

Comment:
Any reason to not just make a more generic `AddReleaseFlags` and put a note in the description of each that they're targeting `fedora` releases?

Reply:
Did this to segregate the flags for `fedora` and `containerlinux`